xchange-stream-dydx/src/main/java/info/bitrich/xchangestream/dydx/dydxStreamingExchange.java

```diff
@@ -0,0 +1,89 @@
+package info.bitrich.xchangestream.dydx;
+
+import info.bitrich.xchangestream.core.ProductSubscription;
+import info.bitrich.xchangestream.core.StreamingAccountService;
+import info.bitrich.xchangestream.core.StreamingExchange;
+import info.bitrich.xchangestream.core.StreamingTradeService;
+import io.reactivex.Completable;
+import org.knowm.xchange.ExchangeSpecification;
+import org.knowm.xchange.dydx.dydxExchange;
+import org.knowm.xchange.exceptions.NotYetImplementedForExchangeException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Author: Max Gao (gaamox@tutanota.com) Created: 20-02-2021
+ *
+ * <p>V3 Documentation: https://docs.dydx.exchange
+ *
+ * <p>Legacy Documentation: https://legacy-docs.dydx.exchange
+ */
+public class dydxStreamingExchange extends dydxExchange implements StreamingExchange {
+  private static final Logger LOG = LoggerFactory.getLogger(dydxStreamingExchange.class);
+  private static final String API_URI_V1 = "wss://api.dydx.exchange/v1/ws";
+  private static final String API_URI_V3 = "wss://api.dydx.exchange/v3/ws";
```

Comment:
Is there a reason you want to build support for v1?

Reply:
v3's orderbook stream only provides data for the 3 perpetual products that dydx currently offers. To access spot and margin market orerbooks, the v1 websocket api is required. I'll get around to adding that in a separate commit.